docs/os/windows/windows-tools41.md

```diff
@@ -0,0 +1,297 @@
+-   User profiles are moved to the private disk on the next reboot after the private disk is initialized. 
+Reboot is required because the "mover utility" runs very early in the boot process so OS can't yet lock any files in there. 
+This can take some time depending on the profiles' size and because the GUI agent is not yet active dom0/Qubes Manager may complain that the AppVM failed to boot. 
+That's a false alarm (you can increase the AppVM's default boot timeout using `qvm-prefs`), the VM should appear "green" in Qubes Manager shortly after.
+
+It also makes sense to disable Automatic Updates for all the template-based AppVMs -- of course this should be done in the Template VM, not in individual AppVMs, because the system-wide settings are stored in the root filesystem (which holds the system-wide registry hives). Then, periodically check for updates in the Template VM and the changes will be carried over to any child AppVMs.
+
+Once the template has been created and installed it is easy to create AppVMs based on it, by selecting the type "AppVM" and a suitable template.
+
+Components
+----------
+
+Qubes Windows Tools (QWT) contain several components than can be enabled or disabled during installation:
+
+- Shared components (required): common libraries used by QWT components.
+- Xen PV drivers: drivers for the virtual hardware exposed by Xen.
+   - Base Xen PV Drivers (required): paravirtual bus and interface drivers.
+   - Xen PV Disk Drivers: paravirtual storage drivers.
+   - Xen PV Network Drivers: paravirtual network drivers.
+- Qubes Core Agent: qrexec agent and services. Needed for proper integration with Qubes.
+   - Move user profiles: user profile directory (`C:\users`) is moved to VM's private disk backed by `private.img file` in `dom0` (useful mainly for HVM templates).
+- Qubes GUI Agent: video driver and gui agent that enable seamless showing of Windows applications on the secure Qubes desktop.
+- Disable UAC: User Account Control may interfere with QWT and doesn't really provide any additional benefits in Qubes environment.
+
+> **Note:** *In testing VMs only* it's probably a good idea to install a VNC server before installing QWT. If something goes very wrong with the Qubes gui agent, a VNC server should still allow access to the OS.
+
+> **Note**: Xen PV disk drivers are not installed by default. This is because they seem to cause problems (BSOD = Blue Screen Of Death). We're working with upstream devs to fix this. *However*, the BSOD seems to only occur after the first boot and everything works fine after that. **Enable the drivers at your own risk** of course, but we welcome reports of success/failure in any case (backup your VM first!). With disk PV drivers absent `qvm-block` will not work for the VM, but you can still use standard Qubes inter-VM file copying mechanisms.
+
```

Comment:
that section is redundant ; remove it altogether ?

Reply:
I merged it with the list at the start of the file.